src/App.js

```diff
@@ -1,41 +1,47 @@
-import React from 'react';
-import { BrowserRouter} from 'react-router-dom'; 
-import { HashLink as Link } from 'react-router-hash-link';
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import { HashLink as Link } from "react-router-hash-link";
 
-import Directory from './portfoliodirectory.js';
-import Footer from './footer.js';
+import Directory from "./portfoliodirectory.js";
+import Footer from "./footer.js";
 
-import './App.css';
-import Logos from './logoreel.js';
+import "./App.css";
+import Logos from "./logoreel.js";
 
 function App() {
   return (
     <BrowserRouter>
       <div className="main">
-      <div id= "home">
-        <div className="navigation">
-            <Link to="/pathLink#home" className="item">home</Link>
-            <Link to="/pathLink#projects" className="item">projects</Link>
-            <Link to="/pathLink#contact" className="item">contact</Link>
-            <div className="animation start-home"></div>
+        <div id="home">
+          <div className="navigation-container">
+            <div className="navigation">
+              <Link to="/pathLink#home" className="item">
+                home
+              </Link>
+              <Link to="/pathLink#projects" className="item">
+                projects
+              </Link>
+              <Link to="/pathLink#contact" className="item">
+                contact
+              </Link>
+              <div className="animation start-home"></div>
             </div>
+          </div>
         </div>
-        <br></br>
-        <br></br>
-        <br></br>
-        <br></br>
-        <div id= "projects">
-        <Directory />
+        <br />
```

Comment:
`<br />` is better than an empty tag but never use `<br>` in react! Swap them all out for margin/padding etc